<launch>
	<arg name="camera_id" />

	<node pkg="ar_sys" type="multi_boards" name="camera$(arg camera_id)" ns="panopticon" output="log" >
		<remap from="/camera_info" to="/usb_cam$(arg camera_id)/camera_info" />
		<remap from="/image" to="/usb_cam$(arg camera_id)/image_raw" />

		<param name="image_is_rectified" type="bool" value="false"/>

		<param name="boards_config" type="string" value="$(find panopticon)/config/boards.yml"/>
		<param name="boards_directory" type="string" value="$(find panopticon)/config/boards"/>

		<param name="draw_markers" type="bool" value="true" />
		<param name="draw_markers_cube" type="bool" value="true" />
		<param name="draw_markers_axis" type="bool" value="true" />
	</node>
</launch>	